enterprise/frontend/src/embedding-sdk/components/public/NewQuestion/NewQuestion.tsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from "react";
+import { usePreviousDistinct } from "react-use";
+
+import {
+  Notebook,
+  QuestionVisualization,
+} from "embedding-sdk/components/public/InteractiveQuestion/components";
+import {
+  InteractiveQuestionProvider,
+  useInteractiveQuestionContext,
+} from "embedding-sdk/components/public/InteractiveQuestion/context";
+import {
+  SaveQuestionForm,
+  SaveQuestionTitle,
+} from "metabase/components/SaveQuestionForm";
+import { SaveQuestionProvider } from "metabase/components/SaveQuestionForm/context";
+import { useCreateQuestion } from "metabase/query_builder/containers/use-create-question";
+import { useSaveQuestion } from "metabase/query_builder/containers/use-save-question";
+import { Button, Group, Stack, Tabs, Title, Box } from "metabase/ui";
+
+const SaveQuestion = ({ onClose }: { onClose: () => void }) => {
+  const { question } = useInteractiveQuestionContext();
+
+  const handleCreate = useCreateQuestion();
+  const handleSave = useSaveQuestion();
+
+  if (!question) {
+    return null;
+  }
+
+  return (
+    <SaveQuestionProvider
+      question={question}
+      originalQuestion={null}
```

Comment:
Should we pass `originalQuestion` that comes from `useInteractiveQuestionContext` as well, or is that not necessary?

Reply:
that actually might be the key for creating `EditQuestion`! nice catch